packages/bautajs-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axa/bautajs-core",
-  "version": "3.0.1",
```

Comment:
please, the package.json version is something managed by the CICD; revert this change.